networkx/algorithms/assortativity/mixing.py

```diff
@@ -117,7 +128,9 @@ def degree_mixing_dict(G, x="out", y="in", weight=None, nodes=None, normalized=F
     return mixing_dict(xy_iter, normalized=normalized)
 
 
-def degree_mixing_matrix(G, x="out", y="in", weight=None, nodes=None, normalized=True):
+def degree_mixing_matrix(
+    G, x="out", y="in", weight=None, nodes=None, mapping=None, normalized=True
```

Comment:
The ordering of the kwargs is tricky here: on the one hand, backwards compatibility policy dictates that new kwargs must be added to the end of the signature; OTOH it's clear that the motivation in this case was to match the kwarg ordering in e.g. `attribute_mixing_matrix`.

My personal preference would be to preserve backward compatibility (i.e. put mapping last) then fix up the ordering once we introduce keyword-only arguments in the future (as our Python version support policy allows).